src/core/Core.js

```diff
@@ -218,7 +218,7 @@ class Uppy {
     this.uploaders.push(fn)
   }
 
-  removeUploader (fn) {
+  _removeUploader (fn) {
```

Comment:
`removeUploader` is used in upload plugins, so it should be public (similar to `removePreProcessor`)